src/praisonai/praisonai/gateway/kanban_dispatcher.py

```
@@ -34,7 +34,26 @@ def __init__(self, max_concurrent: int = 3, poll_interval: float = 5.0):
+            store.close_run(run_id, outcome, summary=summary, metadata=metadata, error=error)
+        except Exception as e:
+            logger.warning(f"Failed to close run {run_id} ({outcome}): {e}")
+
+    def _record_failure_safe(self, store, task_id, *, error=None) -> bool:
+        """Record a failure / circuit-breaker check, swallowing errors.
+
+        Returns True if the task was auto-blocked.
+        """
+        try:
+            return store.record_failure(task_id, error=error)
+        except Exception as e:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)